src/api/searchApi.ts

```diff
@@ -552,3 +553,40 @@ export const fetchResults = async (params: FetchResultsParams, signal?: AbortSig
 
   return getResults.data;
 };
+
+export enum CustomerResultParam {
+  Status = 'status',
+}
+
+export interface FetchCustomerResultsParams {
+  [ResultParam.From]?: number | null;
+  [ResultParam.Order]?: ResultSearchOrder | null;
+  [ResultParam.Query]?: string | null;
+  [ResultParam.Results]?: number | null;
+  [ResultParam.Sort]?: SortOrder | null;
+  [CustomerResultParam.Status]?: RegistrationStatus | RegistrationStatus[] | null;
```

Comment:
fjern til det ev. trengs
```suggestion
  [CustomerResultParam.Status]?: RegistrationStatus[] | null;
```